src/server/appserver.cpp

```diff
@@ -122,10 +122,18 @@ AppServer::AppServer(int &argc, char **argv) :
     setApplicationVersion(_theme->version());
 
     // Setup logging with default parameters
-    initLogging();
+    if (!initLogging()) {
+        throw std::runtime_error("Unable to init logging.");
+    }
+
+    if (isRunning()) {
+        LOG_INFO(_logger, "AppServer already running");
+        return;
+    }
```

Comment:
So we will not parse the arguments anymore if the app is already running? There is a few command that can be run in terminal like "--help", "--version" or "--settings"